things/api.py

```diff
@@ -286,7 +290,7 @@ def tags(title=None, include_items=False, **kwargs):
         If no path is provided, then access the default database path.
 
     database : things.database.Database, optional
-        Any valid `things.database.Database` object previously instantiated.
```

Comment:
no link to the Database definition?

Reply:
@AlexanderWillner removed because it's already defined one line above. That one has a link.